### ISSUE TITLE
plugin issue

### ISSUE THREAD

**Issue by NyakudyaA** [ruz76/qgis_time2web#1]
if the plugin is derived from qgis2web plugin, what makes it unique. If you have any features that the plugin does which are different from  qgis2web plugin why not consider merging your work into  qgis2web plugin

**Comment by tomchadwin** [comment]
It has been merged into qgis2web: https://github.com/tomchadwin/qgis2web/pull/531

**Comment by tomchadwin** [comment]
I think this repo is no longer maintained, as @ruz76 moved it to https://github.com/ruz76/qgis2web_time_extension.

**Comment by NyakudyaA** [comment]
ok thanks @tomchadwin  was just asking because I see it in the plugin repo as unapproved. So I assume it is safe to delete(qgis_time2web)  from qgis plugin repo

**Comment by ruz76** [comment]
Yes. You can delete qgis_time2web from qgis plugin repo. I will continue with extension for @tomchadwin plugin.
